cli/src/pcluster/aws/aws_resources.py

```diff
@@ -309,7 +309,8 @@ def snapshot_ids(self) -> list:
         """Return snapshot ids."""
         snapshot_ids = []
         for block_device_mapping in self.block_device_mappings:
-            snapshot_ids.append(block_device_mapping.get("Ebs").get("SnapshotId"))
+            if block_device_mapping.get("Ebs"):
```

Comment:
I think you should use: `if block_device_mapping.get("Ebs", None):`to cover the case on which Ebs is not defined.

Reply:
Isn't `if block_device_mapping.get("Ebs", None)` equal to `if block_device_mapping.get("Ebs")`

Reply:
You're right! None is the default returned value.